test/ao/sparsity/test_data_scheduler.py

```diff
@@ -38,12 +44,14 @@ def _get_data(self):
         ]
         return data_list, defaults, data_with_config
 
-    def _get_sparsifier(self):
+    def _get_sparsifier(self, return_data=False):
```

Comment:
nit: if we pass around data in this api as suggested by the last PR we probably do not need this argument